modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/feature_indicator/FeatureIndicator.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayBadge from '@clayui/badge';
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayPopover, {ALIGN_POSITIONS} from '@clayui/popover';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import React, {useState} from 'react';
+
+import useId from '../hooks/useId';
+import LearnMessage, {
+	LearnResourcesContext,
+} from '../learn_message/LearnMessage';
+
+export type DisplayType =
+	| 'primary'
+	| 'secondary'
+	| 'success'
+	| 'warning'
+	| 'danger'
+	| 'info'
+	| 'beta'
+	| 'beta-dark'
+	| 'deprecated';
```

Comment:
I'm not aware of the detailed requirements for the FeatureIndicator, nevertheless, I wonder if component needs to expose this amount of DisplayTypes. 

My main concern is not just the amount but rather the intent: for the feature indicator component, the status of the feature should be the main API element. Such status would then be translated to the corresponding display types provided by the design system to represent it visually.

As of now, I can think of `beta` and `deprecated` for these, though there may be more like `new` or variations of the above.

What's makes thing a bit confusing to me is the mix of both feature statuses and display types on the same. Also, I see we have a similar situation in clay-badge component. So at this point we perhaps can consider reusing the [badge displaytypes](https://github.com/liferay/clay/blob/77162efe62cd3822f4a64e23b3469e46cee5c530/packages/clay-badge/src/index.tsx#L9-L17) instead.

Makes sense?

Reply:
Thanks @dsanz that makes a lot of sense. I think we can definitely simplify this API to make it easier for teams to use it correctly.

Reply:
@dsanz I updated DisplayType in https://github.com/liferay-frontend/liferay-portal/pull/3900. I couldn't push to @ethib137 repo.

I also replaced/removed BetaButton and BetaBadge with FeatureIndicator.

Reply:
I'll add a few more updates and push them here.